main.py

```diff
@@ -0,0 +1,50 @@
+import random
+import sys
+from PIL import Image
+from pathlib import Path
+
+VALID_IMAGES = [".jpg", ".jpeg", ".png"]
+
+
+def get_images(location: str) -> list:
+    images = []
+    directory = Path(location)
+    try:
+        for file in directory.iterdir():
+            ext = Path(file).suffix
+            if file.is_file() and ext in VALID_IMAGES:
+                images.append(file)
+    except FileNotFoundError:
+        print("No such file exists")
+        raise
+    finally:
```

Comment:
finally? should this happen in case of error?

Reply:
yes. if no error occurs, run as usual. if error occurs, return empty array and handle in main

Reply:
You reraise in the except block and then return in the finally block. The error isn't raised this way. Furthermore, this makes how the program will act ambiguous and is a very bad practice.
If you want to return an empty list in case of an error do so in the except list

Reply:
removed finally and changed error handling